stories/assets/js/variables.js

```diff
@@ -0,0 +1,12 @@
+const BREAKPOINT = window.BREAKPOINT || {};
```

Comment:
@neelesh-tippe Kindly change this to something more robust. 
Eg.
Change the filename to something like `global-variables.js`. And call the variables something like this:
```
// In global-variables.js

// Define functions which can be called globally.
const foo = (x) => { 
  return x; 
}
const bar = (y) => {
  return y;
}
const baz = (z) => {
  return z;
}

// Define Global Breakpoints in Js.
const breakpoints = {
  small: 767,
  medium: 768,
  large: 1024,
  extraLarge: 1440,
  mediumTab: 1439,
  tablet: 834,
  ...
};

export {foo, bar, baz, breakpoints};
----

// In your Js file.
import { foo, breakpoints } from './global-variables';
...
...
...
console.log(foo(5)); // 5
console.log(breakpoints.small); // 767
console.log(breakpoints.medium); // 768
```